app/app.go

```diff
@@ -19,6 +19,10 @@ func Handle(repoList map[string]string, projectTypeToCheck string, filter string
 			output_channel <- output.Print(fmt.Sprintf("\nChecking '%s' (%s) repos...", projectType, path))
 
 			visit := func(visitedPath string, info os.FileInfo, err error) error {
+				if err != nil {
+					fmt.Printf("error: %v\n", err)
```

Comment:
Theres already a channel for outputting errors, could you change it to something like this:

https://github.com/stevenjack/cig/blob/master/app/app.go#L32

That'll automatically output the error and exit with a non zero.

Thanks for the PR!